seqr/views/utils/dataset_utils.py

```diff
@@ -472,19 +479,19 @@ def update_sample_models():
         for guid in to_delete_sample_individuals:
             existing_samples_by_guid[guid]['dataSource'] = data_source
 
-    def save_sample_data(sample_guid, sample_data):
+    def save_sample_data(sample_key, sample_data):
         if not sample_data:
             return
 
-        sample_guids_to_load.add(sample_guid)
-        save_data(sample_guid, sample_data)
+        save_data(sample_key, sample_data)
 
     def get_matched_sample(sample_key, unmatched_samples, sample_id_to_individual_id_mapping):
```

Comment:
Since this fn no longer returns anything, I don't think it should be prefixed by 'get'. Can you rename it?